### ISSUE TITLE
Description not showing on show page

### ISSUE THREAD

**Issue by hackmastera** [pulibrary/figgy#338]
When I add a description via a script like this:

```
ScannedMap.new(title: Faker::Space.star, description: Faker::Matz.quote, ...)
````

I do not see that description anywhere on the show page. I do see it when I look at the edit page. It's in `[]`. Screenshot:
<img width="1126" alt="screen shot 2017-10-09 at 3 10 04 pm" src="https://user-images.githubusercontent.com/845363/31354337-0a194c8c-ad04-11e7-9063-4ad0009868f2.png">

**Comment by hackmastera** [comment]
"That looks like it needs `multiple: false` in the ScannedMapChangeSet, and maybe `change_set.prepopulate!` isn’t being called." - @tpendragon

**Comment by jrgriffiniii** [comment]
In response to the request to set `multiple:false` for `description` within `ScannedMapChangeSet`, please note that multiple descriptions are currently supported in Plum:
https://hydra-dev.princeton.edu/concern/raster_works/new?locale=en

**Comment by hackmastera** [comment]
Sounds like that would cause a migration / data problem then.

**Comment by hackmastera** [comment]
partially resolved in #359 but description still shows in `[]` on scannedmap edit form.
